addons/binding/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/protocol/xml/XMLConnection.java

```diff
@@ -151,12 +161,55 @@ public String sendReceive(String message) throws IOException {
         }
     }
 
+    private Charset getResponseCharset(HttpURLConnection connection, Charset defaultCharset) {
+
+        // See https://stackoverflow.com/a/3934280/1906057
+
+        Charset charset = defaultCharset;
+
+        String contentType = connection.getContentType();
+        String[] values = contentType.split(";"); // values.length should be 2
+        String charsetName = "";
+
+        // Example:
+        // Content-Type:text/xml; charset="utf-8"
```

Comment:
To be a normal valid header a space seems to be missing.

Reply:
This was copied from my AVR response using Chrome dev tool bar. Apparently space is not required according to the [protocol spec](https://tools.ietf.org/html/rfc2616#section-4.2). Also this is just a comment :)